modules/core/src/copy.cpp

```diff
@@ -285,6 +285,8 @@ void Mat::copyTo( OutputArray _dst ) const
 
         if( rows > 0 && cols > 0 )
         {
+            if (_dst.isVector())
+                dst = dst.reshape(0, dst.cols);
```

Comment:
such call of reshape gives valid results as long as dst is 1xN before the call.
I suggest to change it to more robust
<pre>
dst = dst.reshape(0, dst.cols*dst.rows);
</pre>